Code/Editor/EditorPreferencesDialog.cpp

```diff
@@ -39,7 +39,7 @@ AZ_PUSH_DISABLE_DLL_EXPORT_MEMBER_WARNING
 AZ_POP_DISABLE_DLL_EXPORT_MEMBER_WARNING
 
 using namespace AzQtComponents;
-
+#pragma optimize("",off)
```

Comment:
This should be removed before submission

Reply:
Oopps, thought I'd checked! Done.